CovertInfiltration/Src/CovertInfiltration/Classes/X2Helper_Infiltration.uc

```diff
@@ -11,6 +11,10 @@ class X2Helper_Infiltration extends Object config(Infiltration);
 
 var config int PERSONNEL_INFIL;
 var config int PERSONNEL_DETER;
+
+var config int BASE_INTEL_COST;
+var config int INTEL_COST_MULTIPLIER;
```

Comment:
Could you prepend `EXFILTRATION_` to the variable names? It's a bit unclear what exactly they are used for right now